Packs/Phishing/Scripts/FindDuplicatePhishingIncidents/FindDuplicateEmailIncidents.py

```diff
@@ -225,11 +225,11 @@ def return_entry(message, duplicate_incidents_df=None, similarity=0):
             'id': most_similar_incident['id'],
             'name': most_similar_incident.get('name'),
             'similarity': similarity,
-            'allSimilarIncdientsIds': duplicate_incidents_df['id'].tolist()
         }
     outputs = {
         'similarIncident': similar_incident,
-        'isSimilarIncidentFound': duplicate_incidents_df is not None
+        'isSimilarIncidentFound': duplicate_incidents_df is not None,
+        'allDuplicateIncidentsIds': duplicate_incidents_df['id'].tolist() if duplicate_incidents_df is not None else []
```

Comment:
I think you need to return more info about the incidents, maybe preform the format function to all the list of the incidents and return it as "allDuplicateIncidents" or "similarIncidentList".
If we are going to use this as sub-script for the campain we will need to more info to display.
WDYT?